spec/lib/govuk_logging_spec.rb

```diff
@@ -71,22 +86,60 @@ def app
         Rails.application
       end
 
-      it "logs errors thrown by the application" do
+      it "logs errors thrown by the application with govuk_request_id" do
+        stub_const("GdsApi::GovukHeaders", GovukHeaders)
```

Comment:
```suggestion
        stub_const("GdsApi::GovukHeaders", govuk_headers_class)
```

We can skip the intermediary step.